src/main/java/frc/robot/commands/AutonModes.java

```diff
@@ -19,7 +20,7 @@
 import java.io.IOException;
 import java.nio.file.Path;
 
-public class AutonModes {
+public class AutonModes extends SubsystemBase {
```

Comment:
Why do we want it to be a subsystem?

Reply:
I'll fix that, it was a discontinuous idea.